### ISSUE TITLE
I have a question in "Measurement Update Derivations » Feature Triangulation"

### ISSUE THREAD

**Issue by atkai** [rpng/open_vins#405]
<img width="827" alt="屏幕截图 2023-12-09 145816" src="https://github.com/rpng/open_vins/assets/82944632/7600659d-5c68-4e2e-9e85-621b728d0aaf">

The variable {}^{A}b_{i} should be {}^{A}p_[C_i] ？Or there were some details I ignored?

**Comment by goldbattle** [comment]
Yes, thanks! Looks like a bad copy paste. You can refer to equation above this for what it should be:
![image](https://github.com/rpng/open_vins/assets/2222562/a0a953b7-9965-4fb8-8242-b703eb935518)

**Comment by atkai** [comment]
Thank you! I have no other questions now.
